### ISSUE TITLE
Add serial interface to spin motor

### ISSUE THREAD

**Issue by brycedjohnson** [betaflight/betaflight-esc#20]

**Comment by jaxxzer** [comment]
Is the aim to use UART hardware?

**Comment by brycedjohnson** [comment]
Yes, it would be really handy to have that as an option for testing. Maybe
even a replacement for dshot/oneshot on certain rigs.

**Comment by jaxxzer** [comment]
Are there any opinions on a format currently? Something following an existing protocol? Plain text for the arduino crowd?

**Comment by brycedjohnson** [comment]
For now there is the basic CLI similar to betaflight world be nice to use
(acsii) for testing.  I put a couple functions in there and a motor test
would be great as well. Maybe command to 0 to 100 throttle or tell it to go
to a set RPM etc...

A compact binary serial protocol would be down the road if it was to be a
dshot/multishot replacement. That was just an idea I'd like to test out at
some point to see if it would be fast enough.

Bryce
